portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,17 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="background"></div>
+      <div id="home-content">
+        <h1>Welcome to my website!</h1>
+        <hr>
+        <h5>Where would you like to go?</h5>
+        <ul>
+            <li> About </li>
+            <li> Achievements </li>
+            <li> Contact </li>
```

Comment:
This should actually link out to your new page right?

Reply:
I add the links in the pull request after add_header